docs/components/InputFile/InputFile.stories.mdx

```diff
@@ -58,3 +58,33 @@ updating an array of `FileUpload`s based on their `key` as uploads progress.
  */
 export function updateFiles(updatedFile: FileUpload, files: FileUpload[]);
 ```
+
+### validator
+
+InputFile can accept a `validator` function as a prop. This will allow you to
+reject or accept file uploads based on your needs. The value of the function
+must accept a File and return `null` when it is accepted or an error object when
+rejected.
+
+For example, the code below only accepts files that are less than 2MB in size.
+
+```ts
+function pngFileValidator(file: File) {
```

Comment:
this function name is still referencing the png validation, let's update that!

Reply:
renamed